ci/Jenkinsfile-linux-integration

```diff
@@ -234,11 +234,11 @@ builders['linux-tests'] = testBuilder('linux', 'py35', '/workspace')({
                export CLI_TEST_SSH_USER=centos; \
                export CLI_TEST_MASTER_PROXY=true; \
                dist/dcos cluster setup dcos.snakeoil.mesosphere.com \
-                   --insecure --username=${DCOS_ADMIN_USERNAME} \
-                   --password-env=DCOS_ADMIN_PASSWORD; \
+                  --insecure --username=${DCOS_ADMIN_USERNAME} \
+                  --password-env=DCOS_ADMIN_PASSWORD; \
                dist/dcos config set core.reporting false; \
                dist/dcos config set core.timeout 5; \
-               make test-binary'''
+               make binary'''
```

Comment:
You can't just run `make binary` here or it won't run the integration tests (it will just build the binary).

Reply:
this is still wip, I've changed this so that builds take less time and I can iterate faster, I'll revert it when they pass.